nucleus/core/bootstrap/src/main/java/com/sun/enterprise/glassfish/bootstrap/GlassFishMain.java

```diff
@@ -290,10 +288,10 @@ private List<String> parseCommand(String line) {
             }
             return tokens;
         }
- 
+
         /**
          * Runs a series of commands from a file
-         * @param file 
+         * @param file
          */
         private void doBootCommands(String file) {
```

Comment:
`doBootCommands` function can be removed as no more used in this class.

Reply:
It still used for `-prebootcommandfile` execution at line 138